lib/reports/etas_organization_overlap_report.rb

```diff
@@ -22,6 +22,10 @@ def initialize(organization = nil)
       # public access location
       @remote_report_path = Settings.etas_overlap_reports_remote_path
       @organization = organization
+
+      # TODO - this probably won't scale; would be better to iterate over
```

Comment:
I suspect this will have a similar issue to what we see with the concordance validation, especially if we try to run it for all holdings